.travis.yml

```diff
@@ -71,7 +72,25 @@ before_install:
   - composer self-update --1
 
 install:
+  - |
+    composer config repositories."1"  "{
+      \"type\": \"package\",
+      \"package\": {
+        \"name\": \"gravityforms/gravityforms\",
+        \"version\": \"2.4.21.3\",
+        \"type\": \"wordpress-plugin\",
+        \"dist\": {
+          \"type\": \"zip\",
+          \"url\": \"https://www.gravityhelp.com/wp-content/plugins/gravitymanager/api.php?op=get_plugin&slug=gravityforms&key=$GRAVITYFORMS_KEY\"
+        },
+        \"require\": {
+          \"composer/installers\": \"^1.4\",
+          \"gotoandplay/gravityforms-composer-installer\": \"^2.3\"
+        }
+      }
+    }"
   - npm install
+  - composer require --dev gravityforms/gravityforms:2.4.21.3
```

Comment:
@kasparsd This should do the trick. I don't think there is a need for an alternative `composer.json`, but it does make the repositories array and object.
![image](https://user-images.githubusercontent.com/13604318/109582155-a6c7e500-7acb-11eb-82b1-5beaac2c4062.png)
But I revert it in the `before_deploy` :thinking:.